content/en/docs/ops/configuration/security/security-policy-examples/index.md

```diff
@@ -76,10 +84,19 @@ spec:
 
 ### Namespace isolation with ingress exception
 
-The following policy allows traffic only from the namespace `foo` and also from the ingress gateway. This requires you have already enabled mTLS.
+The following two policies enable strict mTLS and allow traffic from the namespace `foo` and also from the ingress gateway.
```

Comment:
enable strict mTLS on namespace 'foo', and allow traffic from the same namespace and also from the ingress gateway.